tests/test_client/tests.py

```diff
@@ -595,6 +595,14 @@ def test_external_redirect(self):
         response = self.client.get('/django_project_redirect/')
         self.assertRedirects(response, 'https://www.djangoproject.com/', fetch_redirect_response=False)
 
+    def test_external_redirect_with_fetch_error_msg(self):
+        "Regression for https://code.djangoproject.com/ticket/26666"
```

Comment:
No need for this comment unless the ticket has some information that's not straightforward.